wled00/set.cpp

```diff
@@ -587,6 +587,8 @@ void handleSettingsSet(AsyncWebServerRequest *request, byte subPage)
       wifiLock = request->hasArg(F("OW"));
       aOtaEnabled = request->hasArg(F("AO"));
       //createEditHandler(correctPIN && !otaLock);
+      interfaceUpdateCallMode = CALL_MODE_WS_SEND;
```

Comment:
Do you think it is really necessart to broadcast state/info after settings are saved?
IMO not but I may be missing a usecase.

Reply:
Yes it is. If this PR is that you know if the settings value, then you need to send the new value when you save

Reply:
Indeed, it would allow an app using websockets to update its state right away